GameDataParser/Parsers/ItemParser.cs

```diff
@@ -518,23 +415,21 @@ protected override List<ItemMetadata> Parse()
         return rewards;
     }
 
-    private static void ParseHair(XmlNode slot, ItemMetadata metadata)
+    private static void ParseHair(Slot slot, ItemMetadata metadata)
     {
-        int assetNodeCount = slot.SelectNodes("asset").Count;
-        XmlNode asset = slot.FirstChild;
-
-        XmlNode scaleNode = slot.SelectSingleNode("scale");
+        int assetNodeCount = slot.asset.Count;
+        Slot.Scale scaleNode = slot.scale?.FirstOrDefault();
 
         CoordF defaultCoord = CoordF.Parse("0, 0, 0");
         switch (assetNodeCount)
```

Comment:
Would be more clear to use `slot.asset.Count` here because you are accessing from `slot.asset[?]` below.